### ISSUE TITLE
Middle pane split since 0.15 insider update

### ISSUE THREAD

**Issue by akadormouse** [liamcain/obsidian-creases#31]
Since the 0.15 & 0.15.1 insider updates, Creases causes a horizontal split in the middle pane. Even when no file is open. This is tested with default theme and turning Creases on and off & Restricted mode on and off.
![Snag_23e15772](https://user-images.githubusercontent.com/91322324/174163580-055c2ce1-a828-4a27-bf7f-a595693e7ac1.png)

**Comment by bagameow** [comment]
Yes, I got the same result. It took time to filter which plugin caused this.
